webpack.config.js

```diff
@@ -40,7 +40,7 @@ const assetExporter = (regex, folder) => ({
 			options: {
 				outputPath: folder,
 				name: "[name].[ext]",
-				publicPath: path.join(publicPath, `/${folder}`)
+				publicPath: isProd ? `${publicPath}/${folder}` : path.join(publicPath, folder)
```

Comment:
what's the difference here?

Reply:
path join break http urls
```js
> path.join('http://whate.ver', 'blah')
'http:/whate.ver/blah'
```

Reply:
ohh ok then let's remove `path.join` and only use the first version